### ISSUE TITLE
Negotiation Goods Choice [wish list]

### ISSUE THREAD

**Issue by ljettinger** [dsiekiera/foe-helfer-extension#488]
I've noticed while using the Neg Helper that when it's guessing, it will by default use lower aged goods no matter how many are actually in stock compared to current aged goods... Since the beginning of GBG, on my main world, the "boring skipped ages" goods are in very high demand, so ME, CE and TE, especially TE, so with myself being in FE, the TE goods are very hard to come by and it would be nice to have a checkbox setting either in the settings area or right on the negotiation helper to tell it to use current age, previous age or 'the most of' goods when guessing... There are lots of times when I cringe during GBG when it wants to use 3 guesses with 15ea TE goods that I only have 600 in stock, just to eliminate it from the FE goods that I have 9k in stock...

**Comment by andrgin** [comment]
You can change the priority before starting the negotiation by draging the goods you want to save to the right and the goods you want to spend to the right.

**Comment by ljettinger** [comment]
I was trying to figure out how to change that, I kept trying to move the second row... lol... thanks for that...

**Comment by andrgin** [comment]
There is now a checkbox to disable saving of current era goods.
